src/pcloud/api.py

```diff
@@ -38,9 +38,18 @@ def main():
 
 class PyCloud(object):
 
-    endpoint = "https://api.pcloud.com/"
+    O_WRITE = int("0x0002", 16)
+    O_CREAT = int("0x0040", 16)
+    O_EXCL = int("0x0080", 16)
+    O_TRUNC = int("0x0200", 16)
+    O_APPEND = int("0x0400", 16)
```

Comment:
Please remove the flags here. They are available on top level and need to be imported explicitly, if needed.